### PR TITLE
Removing problem from Bronze section, adding cpp solution + good editorial to usaco.1059 😈 

### PR DESCRIPTION
**Pull request by sathu-beep**
this problem too hard to be taught under the "INTRO TO GREEDY" lol this isn't intro Ill add it to silver greedy section

### CODE REVIEW

**Comment by bqi343** [comment]
silver greedy focuses on sorting, and this problem doesn't involve sorting. so it should either be in this module or no module at all (which might be preferable)

**Comment by sathu-beep** [comment]
yea nathan told me about removing unhelpful problems from bronze/silver and I think this problem def doesn't belong in bronze, esp in the "intro to greedy" since this is p hard. Just removing the problem from the guide for now should be the better solution. Having only bronze in bronze might be a good idea too but the other silver problems in bronze module I haven't looked at yet

**Comment by sathu-beep** [comment]
yo @bqi343 can u merge since I cant make any more prs until this one is merged and I have some that I want to submit by today since starting from tmr I will be busy.

**Comment by sathu-beep** [comment]
Oh what nvm they all just went into this pull request 😒 now the title is inapplicable

File: solutions/usaco-1059.mdx [hunk 1]
```diff
@@ -8,6 +8,46 @@ author: Sathvik Chundru
 [Official Analysis](http://www.usaco.org/current/data/sol_prob1_bronze_dec20.html)
 
 <LanguageSection>
+
+<CPPSection>
+
+```cpp
```
Comment (maggie-j-liu) [inline solutions/usaco-1059.mdx:14]: i think the official analysis is very clear, so this is unnecessary...

File: content/2_Bronze/Ad_Hoc.problems.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "MODULE_ID": "ad-hoc",
   "general": [
-    {
+     {
```
Comment (maggie-j-liu) [inline content/2_Bronze/Ad_Hoc.problems.json:4]: why??

Comment (sathu-beep) [reply content/2_Bronze/Ad_Hoc.problems.json:4]: that was a mistake from an old pr

**Comment by HarryWangATX** [comment]
why can't u just make a new branch?

File: solutions/usaco-1059.mdx (hunk 1 above)
Comment (sathu-beep) [reply solutions/usaco-1059.mdx:14]: should i just delete it then 
the whole cpp solution

**Comment by thecodingwizard** [comment]
wait this is a silver-level problem though, shouldn't we move it to the conclusion module?

**Comment by HarryWangATX** [comment]
> wait this is a silver-level problem though, shouldn't we move it to the conclusion module?

yes 👍

File: solutions/usaco-1059.mdx (hunk 1 above)
Comment (sathu-beep) [reply solutions/usaco-1059.mdx:14]: ah rip here it goes then

File: content/1_General/Adding_Solution.mdx [hunk 3]
```diff
@@ -99,7 +99,7 @@ See [Working With MDX](/general/working-mdx) for additional information.
 
     - Use _tabs_ to indent
     <Info title="Pro Tip">
-	If your code is indented with spaces, use the [Guide Editor](/editor), copy-paste your code in, right-click, click "Command Palette" from the menu that pops up and then search for "Convert Indentation to Tabs". Once you see it, click on it and then copy that new code with the correct indentation into your Github pull request.
+	If your code is indented with spaces, use the [Guide Editor](/editor). Copy-paste your code in, highlight everything then right-click, click "Command Palette" from the menu that pops up and then search for "Convert Indentation to Tabs". Once you see it, click on it and make sure that all spaces were converted to tabs since certain parts of code can still have bugged spaces. If everything looks good, copy that new code with the correct indentation into your Github pull request.
```
Comment (dolphingarlic) [inline content/1_General/Adding_Solution.mdx:102]: Why was this changed? The original is better imo

**Review by dolphingarlic — CHANGES_REQUESTED** [review]

File: content/1_General/Adding_Solution.mdx (hunk 3 above)
Comment (bqi343) [reply content/1_General/Adding_Solution.mdx:102]: it works for me without highlighting anything

**Comment by sathu-beep** [comment]
closing this pr if its not helpful

**Comment by sathu-beep** [comment]
actually just get rid of the silver thing and once you do I will delete the pr
